install.sh

```diff
@@ -262,17 +263,17 @@ function update_device_json () {
   cd "${CREW_CONFIG_PATH}"
 
   if grep '"name": "'"${1}"'"' device.json > /dev/null; then
-    echo "Updating version number of ${1} in device.json..."
+    echo -e "Updating version number of ${1} in device.json...${RESET}"
     sed -i device.json -e '/"name": "'"${1}"'"/N;//s/"version": ".*"/"version": "'"${2}"'"/'
   elif grep '^    }$' device.json > /dev/null; then
-    echo "Adding new information on ${1} to device.json..."
+    echo -e "${RESET}Adding new information on ${1} to device.json..."
     sed -i device.json -e '/^    }$/s/$/,\
     {\
       "name": "'"${1}"'",\
       "version": "'"${2}"'"\
     }/'
   else
-    echo "Adding new information on ${1} to device.json..."
+    echo -e "${RESET}Adding new information on ${1} to device.json...${RESET}"
```

Comment:
I'm confused.  Why 2 `${RESET}`s at the start and end of the string?

Reply:
Removed